fuels-abigen-macro/tests/harness.rs

```diff
@@ -1292,7 +1293,7 @@ async fn test_gas_errors() {
     // Gas limit will be 100, this call will use more than 100 gas.
     let result = contract_instance
         .initialize_counter(42) // Build the ABI call
-        .tx_params(TxParameters::new(None, Some(100), None))
+        .tx_params(TxParameters::new(None, Some(100), None, None))
```

Comment:
This is definitely a good reason for a builder pattern! Not breaking all downstream.